app/src/main/java/io/neurolab/main/NeuroLab.java

```diff
@@ -269,7 +271,13 @@ public void onBackPressed() {
             drawer.closeDrawer(GravityCompat.START);
         } else {
             unregisterReceiver(broadcastReceiver);
-            super.onBackPressed();
+            if (mBackPressed + TIME_INTERVAL > System.currentTimeMillis()) {
```

Comment:
override onBackPressed instead of editing here

Reply:
@adityastic Will you please elaborate a bit? Should I create a separate function for this snippet?
```
if (mBackPressed + TIME_INTERVAL > System.currentTimeMillis()) {
                super.onBackPressed();
                return;
            } else {
                Toast.makeText(getBaseContext(), R.string.double_tap_back, Toast.LENGTH_SHORT).show();
            }
            mBackPressed = System.currentTimeMillis();
```

Reply:
Oh no. I missed the place, it is inside the correct place. Dw